### PR TITLE
ls: fix size units in example description

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Ref: https://github.com/tldr-pages/tldr/pull/5437#discussion_r593805281

### CODE REVIEW

**Comment by bl-ue** [comment]
Makes sense.  From the BSD (macOS) man page (bold mine):
>Byte, Kilobyte, Megabyte, Gigabyte, Terabyte and Petabyte in order to reduce the number of digits to three or less using **base 2** for sizes

From the GNU man page:
>Units are K,M,G,T,P,E,Z,Y (powers of 1024) or KB,MB,... (powers of 1000).  Binary prefixes can be used, too: KiB=K, MiB=M, and so on.

**Comment by bl-ue** [comment]
Also @navarroaxel will you please update these translations? (There are others, but they either don't have the units or translate the unit letters.)
- [`pages.de/common/ls.md`](https://github.com/tldr-pages/tldr/blob/master/pages.de/common/ls.md)
- [`pages.es/common/ls.md`](https://github.com/tldr-pages/tldr/blob/master/pages.es/common/ls.md)
- [`pages.hi/common/ls.md`](https://github.com/tldr-pages/tldr/blob/master/pages.hi/common/ls.md)
- [`pages.id/common/ls.md`](https://github.com/tldr-pages/tldr/blob/master/pages.id/common/ls.md)
- [`pages.it/common/ls.md`](https://github.com/tldr-pages/tldr/blob/master/pages.it/common/ls.md)
- [`pages.ta/common/ls.md`](https://github.com/tldr-pages/tldr/blob/master/pages.ta/common/ls.md)
- [`pages.pt_BR/common/ls.md`](https://github.com/tldr-pages/tldr/blob/master/pages.pt_BR/common/ls.md)

File: pages.ru/common/ls.md [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 `ls -la`
 
-- Подробный список с выводом размера файла в удобочитаемых единицах (КБ, МБ, ГБ):
+- Подробный список с выводом размера файла в удобочитаемых единицах (КиБ, МиБ, ГиБ):
```
Comment (bl-ue) [inline pages.ru/common/ls.md:21]: Do you know that `и` = `i`?

**Review by sbrl — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]

File: pages.ru/common/ls.md (hunk 1 above)
Comment (bl-ue) [reply pages.ru/common/ls.md:21]: Ah, in fact, [it would appear to](https://github.com/sindresorhus/transliterate/blob/main/replacements.js#L332).

**Review by bl-ue — APPROVED** [review]

File: pages.ru/common/ls.md (hunk 1 above)
Comment (navarroaxel) [reply pages.ru/common/ls.md:21]: Idk, just Wikipedia

Comment (bl-ue) [reply pages.ru/common/ls.md:21]: Yeah, it's fine.  If a Russian doesn't like it, they can come along and fix it if they want to.

**Pull request merged**